content/en/blog/_posts/the-case-for-kubernetes-limits/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "The Case for Kubernetes Resource Limits: Predictability vs. Efficiency"
+date: TODO
+slug: the-case-for-kubernetes-resource-limits-predictability-vs-efficiency
+---
+
+**Author:** Milan Plžík
```

Comment:
```suggestion
**Author:** Milan Plžík (Grafana Labs)
```

If you are not happy to blog as someone who works at Grafana, you could use “independent” instead.

Reply:
I'm definitely okay with that, I just didn't know the affiliation also goes to the Author field :)